website/docs/error-reference.md

```diff
@@ -670,6 +670,26 @@ parameter, while the RBS signature declares it as a positional parameter:
 def method(param:); end
 ```
 
+## 3557
+
+This error is raised when a class or module is annotated with multiple RBS
+generic signature comments:
```

Comment:
wording: is this technically gated on the "signatures" or "assertions" flag?

In any case, I think it would make more sense to say "generic type parameter comments"

Reply:
It's on `--enable-experimental-rbs-signatures`.